io/plugins/eu.esdihumboldt.hale.io.topojson/src/eu/esdihumboldt/hale/io/topojson/TopoJsonInstanceWriter.java

```diff
@@ -268,11 +273,11 @@ public static JsonNode replaceUnwantedCharacters(JsonNode node) {
 								}
 								else {
 									// Create a DateTimeFormatter object with
-									// the desired format
-									DateTimeFormatter formatter = DateTimeFormatter
-											.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
+//									// the desired format
+//									DateTimeFormatter formatter = DateTimeFormatter
```

Comment:
Is this no longer done because there can be a date or date and time?

Reply:
Was just wondering why it was removed but still left as comment. I would say to use the original text value is fine and in line with the previous behavior.